NuGet.config

```diff
@@ -3,9 +3,7 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-extensions-cf04410" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-extensions-cf044102/nuget/v3/index.json" />
-    <add key="darc-pub-dotnet-core-setup-57d5bbb" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-57d5bbb5/nuget/v3/index.json" />
-    <add key="darc-pub-dotnet-corefx-282d5b9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-282d5b9f/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-corefx-8a3ffed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-8a3ffed5/nuget/v3/index.json" />
```

Comment:
@dougbu @wtgodbe @JunTaoLuo Do master branches need these specialty feeds?

Reply:
No, these correspond to the 3.1 packages, which should also be removed from this PR in favor of the 5.0 packages